### ISSUE TITLE
Amazon Linux 2 instructions "wrong"

### ISSUE THREAD

**Issue by grahamwoodward** [intel/processwatch#16]
The README says

`sudo yum update
sudo yum install bpftool zlib-devel zlib-static
  elfutils-libelf-devel-static clang cmake3 llvm glibc-static`

but it doesn't recoqnise  elfutils-libelf-devel-static...I had to run

`sudo yum install elfutils-libelf-devel`

I'm running this on
`$ uname -r
6.1.91-99.172.amzn2023.x86_64`

**Comment by matthew-olson-intel** [comment]
Hm... maybe they removed the `-static` version of `libelf`. Can you open a PR?

**Comment by matthew-olson-intel** [comment]
Is this still an issue?

**Comment by matthew-olson-intel** [comment]
Re-reading your original comment, I realized that I was using a woefully out-of-date version of Amazon Linux 2 (2020). Trying 2023.

**Comment by matthew-olson-intel** [comment]
@grahamwoodward You're totally right. They've changed the package name in Amazon Linux 2. I'll fix in the README.
